hutch_python/tests/test_cli.py

```diff
@@ -1,15 +1,16 @@
+import logging
 import os
 import shutil
-import logging
 from pathlib import Path
 
+import IPython
```

Comment:
Is this line used?

Reply:
Ken is out- I'm going to briefly check on this. Is flake8 not configured to look at the tests folder or something?

Reply:
I think it is but I'm guessing it doesn't notice it because it sees `IPython.core.completer` getting used and isn't smart enough to notice that's from the next import not this one?

Reply:
Yep that's exactly what's going on- just finished checking. I'll remove this line and merge (tests pass locally without it)